### PR TITLE
NAS-117869 / 13.0 / NAS-117869 - Fixed disk temperature display

### PR DESCRIPTION
**Pull request by Raz-Dva**
Need  Mini E/X/X+/XL
Go to System -> View Enclosure
Check the Drive Temperatures list display

image

### CODE REVIEW

File: src/app/pages/system/viewenclosure/enclosure-disks/enclosure-disks-mini.component.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<mat-card 
+<mat-card
   class="view-card {{chassis && chassis.front && chassis.front.totalDriveTrays > 7 ? 'mini mini-xl': 'mini'}}"
-  [ngClass]="{'multiple-enclosures': system && system.profile && system.profile.length > 1}" 
-  id="{{currentTab.alias}}" 
+  [ngClass]="{'multiple-enclosures': system && system.profile && system.profile.length > 1}"
+  id="{{currentTab.alias}}"
   (window:resize)="onResize($event)"
   ngClass.gt-md="gt-md"
```
Comment (aKarpovX100) [inline src/app/pages/system/viewenclosure/enclosure-disks/enclosure-disks-mini.component.html:6]: please sort inputs 
with no [ ] - first
with [ ] - then 
with ( ) - last

File: src/app/pages/system/viewenclosure/enclosure-disks/enclosure-disks-mini.component.html [hunk 2]
```diff
@@ -63,8 +63,8 @@ <h3 class="header">Pools</h3>
         <div *ngIf="enclosure" class="overlay status" style="right: 0;">
           <ng-container *ngFor="let dt of enclosure.driveTrayObjects; let i=index">
             <div *ngIf="findDiskBySlotNumber(i + 1)" style="position:absolute; z-index:12; left:12.5%; top:calc({{stackPositions()[i].y}}px + 8px);">
-              <mat-icon 
-                [ngClass]="{'online': findDiskBySlotNumber(i + 1).status == 'ONLINE', 'alert':findDiskBySlotNumber(i + 1).status !== 'ONLINE' && findDiskBySlotNumber(i + 1).status !== 'AVAILABLE'}" 
+              <mat-icon
+                [ngClass]="{'online': findDiskBySlotNumber(i + 1).status == 'ONLINE', 'alert':findDiskBySlotNumber(i + 1).status !== 'ONLINE' && findDiskBySlotNumber(i + 1).status !== 'AVAILABLE'}"
```
Comment (aKarpovX100) [inline src/app/pages/system/viewenclosure/enclosure-disks/enclosure-disks-mini.component.html:67]: sort here as well

Comment (AlexKarpov98) [reply src/app/pages/system/viewenclosure/enclosure-disks/enclosure-disks-mini.component.html:67]: sorry used my old github account 😃

**Review by undsoft — APPROVED** [review]
Everything appears to be working correctly 👍🏻

**Pull request merged**